btr-api/tests/unit/resources/test_requests.py

```diff
@@ -114,7 +120,7 @@ def test_post(app, client, session, jwt, requests_mock, sample_user, test_name,
 
 @pytest.mark.parametrize(
     'test_name, user_type',
-    [('test_post_unauthorized', UserType.USER_COMPETENT_AUTHORITY)],
+    [('test_post_unauthorized', UserType.USER_PUBLIC)],
```

Comment:
This test seems incorrect? It is described as testing unauthorized, but it is actually testing that a basic user can POST so I think this could just be a an added test case to the one above?

Reply:
It was testing unauthorized I just didn't remove the user_type above or below so it wouldn't let me remove it there. Fixed now